<launch>
   <node name="whycon" type="set_axis" pkg="whycon" output="screen">
     <param name="axis" value="vrep"/>
     <param name="inner_diameter" value="0.1"/>
     <param name="outer_diameter" value="0.2"/>
   </node>
</launch>
  
